index.js

```diff
@@ -221,11 +244,16 @@ class MarkdownToolbarElement extends HTMLElement {
   }
 
   connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'toolbar')
+    }
     const fn = shortcut.bind(null, this)
     if (this.field) {
       this.field.addEventListener('keydown', fn)
       shortcutListeners.set(this, fn)
     }
+    const firstTabIndex = document.querySelector('[role="button"][tabindex]')
+    if (firstTabIndex) firstTabIndex.setAttribute('tabindex', '0')
```

Comment:
Not sure if this is the best approach, but not sure of the alternative here.

Reply:
This could be solve by the data attribute as well.